modules/administration-guide/pages/uninstalling-che.adoc

```diff
@@ -21,20 +21,21 @@ To uninstall an instance of {prod} {prod-ver}:
 
 .Procedure
 
-. Obtain the name of the {prod-short} {orch-namespace} (default: `{prod-namespace}`):
+. Obtain the name of the {prod-short} {orch-namespace}. The default is `{prod-namespace}`.
 +
 [subs="+attributes,quotes"]
 ----
 $ {orch-cli} get checluster --all-namespaces \
   -o=jsonpath="{.items[*].metadata.namespace}"
 ----
 
-. Remove the {prod-short} instance from the __<{prod-namespace}>__  {orch-namespace}:
+. Remove the {prod-short} instance from its {orch-namespace}:
 +
 [subs="+attributes,quotes"]
 ----
-$ {prod-cli} server:delete -n __<{prod-namespace}>__
+$ {prod-cli} server:delete -n {prod-namespace} <1>
```

Comment:
Add a comment before the callout for copy and paste friendliness. See: https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/#copy-and-paste-friendly-callouts

```suggestion
$ {prod-cli} server:delete -n {prod-namespace} # <1>
```